app/views/components/page.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.ui.{Page, Paragraph, HyperLink, H1, H2, H3}
+
+@(pge: Page)(implicit messages: Messages)
+
+@for( cmp <- pge.components) {
+  @{
+    cmp match {
+      case p: Paragraph => components.paragraph(p)
+      case l: HyperLink => components.link(l)
+      case h: H1 => components.h1_heading(h)
+      case h: H2 => components.h2_heading(h)
+      case h: H3 => components.h3_heading(h)
+      case x => {x}
```

Comment:
I am not certain what the default action really should be. This is something we should discuss at a later date.